### PR TITLE
Enable Behat coverage

### LINKED ISSUE #84: Enable Behat coverage
Cf. [CoverageContext](https://github.com/api-platform/core/blob/master/features/bootstrap/CoverageContext.php) from api-platform/core

### PR DESCRIPTION
**Pull request by vincentchalamon**
Fix #84

### CODE REVIEW

**Comment by vincentchalamon** [comment]
Could be replaced by [leanphp/behat-code-coverage](https://github.com/leanphp/behat-code-coverage/) (once phpdg supported: https://github.com/leanphp/behat-code-coverage/pull/42)

File: api/symfony.lock [hunk 1]
```diff
@@ -245,18 +257,27 @@
...
         "version": "3.1.0"
     },
     "sebastian/recursion-context": {
         "version": "3.0.0"
     },
+    "sebastian/version": {
+        "version": "2.0.1"
+    },
     "sensiolabs/security-checker": {
         "version": "4.0",
```
Comment (pborreli) [inline api/symfony.lock]: composer json is "sensiolabs/security-checker": "^5.0" so you should update the composer.lock and commit it maybe ?

Comment (vincentchalamon) [reply api/symfony.lock]: This is the version of the recipe, not the package version. Check: https://github.com/symfony/recipes/tree/master/sensiolabs/security-checker/4.0

**Comment by dunglas** [comment]
Thanks @vincentchalamon

**Pull request merged**